### PR TITLE
Only allow jukebox requests to be made if clean.

### PR DESCRIPTION
**Pull request by mstratford**
Fixes UniversityRadioYork/Issues#123

### CODE REVIEW

**Comment by mstratford** [comment]
Tested on Dev, checked a clean song, a song that had recently been played, and an explicit one. Did what it was supposed to :)

File: src/Classes/iTones/iTones_TrackRequest.php [hunk 1]
```diff
@@ -93,7 +93,8 @@ private function canRequestTrack()
      */
     private function trackCanBePlayed()
     {
-        return !(MyRadio_TracklistItem::getIfPlayedRecently($this->track));
+        return (!(MyRadio_TracklistItem::getIfPlayedRecently($this->track)) &&
+               ($this->track->getClean() !== 'n'));
```
Comment (LordAro) [inline src/Classes/iTones/iTones_TrackRequest.php]: excessive number of brackets?

` return !MyRadio_TracklistItem::getIfPlayedRecently($this->track) && $this->track->getClean() !== 'n';` ?

**Comment by mstratford** [comment]
It would appear travis thinks that many more lines have changed in this PR than is actually the case?

**Comment by LordAro** [comment]
odd, maybe it's been updated or something. just looks like one file, fix it here anyway?

**Pull request merged**